CHANGELOG.md

```diff
@@ -16,6 +19,10 @@ Release built: _not released yet_
 - Replaced the `IX_resource_holders_entity_id_resource_entity_id_balance` index with the `IX_resource_holders_resource_entity_id_balance_entity_id` index on the `resource_holders` table.
 - New `outer_object_entity_id` column in the `entities` table, which holds the outer object entity id (e.g resource entity id for vaults and consensus manager entity id for validators).
 - New `receipt_event_emitter_entity_ids` column in the `ledger_transaction_events` table, which holds the emitter entity ids for transaction events.
+- Added a new `entities_by_role_requirement` table that stores information about entities that have ever used a requirement (resource or non-fungible global ID) in their access rules.
```

Comment:
Would this be more consistent if it were called `entities_by_role_requirement_entry_definition` ?

As in the abstraction of the collections rework, it basically is an `entry_definition` table I think?